README.md

```diff
@@ -18,11 +18,40 @@ mbed import mbed-os-example-bootloader
 cd mbed-os-example-bootloader
 ```
 
-## Set up application to be a bootloader
+## Connecting the SD card
+This example uses the **onboard SD card port** for the K64F and the UBLOX_EVK_ODIN_W2.
 
-All supported boards mentioned above are set up to build as a bootloader image. To add support for a new board, you must specify the size of the bootloader.
+For other targets, the following Arduino form-factor SPI pins are used by default:
```

Comment:
"are used" is passive voice. Please change this to "Other targets use the following Arduino form-factor SPI pins by default:" if it keeps the same meaning.